lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -570,17 +568,15 @@ states:
   MITIGATION_01_STRATEGIC_APP_TRIAGE:
     nestedJourney: STRATEGIC_APP_TRIAGE
     exitEvents:
-      next:
+      sessionError:
```

Comment:
I'm not sure if this is correct - do we have a definitive answer for what to do if the action to set up a v2 app session fails?

Reply:
Presuming this is an error response from DCMAW or something blows up in the lambda? then technical error is fine for now